features/checkout/paying_for_order/preventing_to_pay_for_cancelled_order.feature

```diff
@@ -13,7 +13,7 @@ Feature: Preventing to pay for the cancelled order
         And the store allows paying Offline
         And there is a customer "sylius@example.com" that placed an order "#00000022"
 
-    @ui
+    @ui @no-api
```

Comment:
Tried some ways to implement it in API, but it my opinion it would be some kind of cheat. I think it's fully UI feature, becouse we haven't got any endpoint that allows directly user to do such a thing,